packages/vuetify/src/util/helpers.ts

```diff
@@ -506,3 +506,12 @@ export function humanReadableFileSize (bytes: number, binary = false): string {
   }
   return `${bytes.toFixed(1)} ${prefix[unit]}B`
 }
+
+export function camelizeObjectKeys (obj: Record<string, any> | null | undefined) {
+  if (!obj) return {}
```

Comment:
```suggestion
  if (obj == nul) return {}
```

Thoughts on this?

Reply:
double equality will fail if `obj` is an empty string